src/ts-units.ts

```diff
@@ -84,6 +90,16 @@ export class TsParameter {
     }
 }
 
+export class TsOption {
```

Comment:
I'd call this TsEnumEntry or something similar. Option reminds of optional types...